receivers/pushover/pushover.go

```diff
@@ -175,7 +175,11 @@ func (pn *Notifier) genPushoverBody(ctx context.Context, as ...*types.Alert) (ma
 		return nil, b, fmt.Errorf("failed write the message: %w", err)
 	}
 
-	pn.writeImageParts(ctx, w, as...)
+	if pn.settings.Upload {
+		pn.writeImageParts(ctx, w, as...)
```

Comment:
I think we should be doing something with the error currently being swallowed by `writeImageParts()`, at minimum logging it. WDYT?